lib/openstax/salesforce/remote/school.rb

```diff
@@ -20,7 +20,7 @@ class School < ActiveForce::SObject
         self.table_name = 'Account'
 
         def self.query
-          super.where("RecordType.Name = 'School'")
+          super.where("RecordType.Name = 'School' OR RecordType.Name = 'School District'")
```

Comment:
This seems unconventional but I guess it could work if the reason is some legacy code is treating them as the same thing?

Reply:
@Dantemss we have noticed that folks in Accounts are getting assigned to school districts (from SheerID), which are not getting cached in the Accounts DB, so it's causing sync errors.
This works fine locally for what I'm trying to accomplish -- and yes, they are stored in Salesforce under the same objects (just a different record type).

I guess the alternative would be to make a new remote object for School District but that would certainly complicate the Accounts code a bit more.